frontend/src/Pages/ResidentOverview.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+    ResidentProgramClassHistory,
+    ResidentProgramClassInfo,
+    ResidentProgramClassWeeklySchedule,
+    ServerResponseMany
+} from '@/common';
+import GreyPill from '@/Components/pill-labels/GreyPill';
+import { useAuth } from '@/useAuth';
+import useSWR from 'swr';
+
+export default function ResidentOverview() {
+    const { user } = useAuth();
+    const user_id = user?.id;
+    const {
+        data: programsResp
+        // error: programsError,
+        // isLoading
+    } = useSWR<ServerResponseMany<ResidentProgramClassInfo>, Error>(
+        `/api/users/${user_id}/programs?order=ASC&order_by=program_name`
+    );
+    const enrollment_metrics = programsResp?.data;
+
+    const {
+        data: weekly_scheduleResp
+        // error: weekly_scheduleError,
+        // isLoading
+    } = useSWR<ServerResponseMany<ResidentProgramClassWeeklySchedule>, Error>(
+        `/api/users/${user_id}/weekly-schedule`
+    );
+    const weekly_schedule_metrics = weekly_scheduleResp?.data;
+    console.log('weekly_schedule_metrics:>>   ', weekly_schedule_metrics);
+
+    const {
+        data: historyResp
+        // error: historyError,
+        // isLoading
+    } = useSWR<ServerResponseMany<ResidentProgramClassHistory>, Error>(
+        `/api/users/${user_id}/program-history`
+    );
```

Comment:
@corypride We won't need this if we consolidate the call to one handler, though there will be some work on the front-end to split it between active classes and historical data. You could probably put it in a map and then render the two different parts on the page using that data.